### PR TITLE
Document mysqli_sql_exception::getSqlState()

### PR DESCRIPTION
**Pull request by kocsismate**
I have to admit that the descriptions are not too informative.

### CODE REVIEW

**Review by cmb69 — COMMENTED** [review]
Thank you! This looks good to me, but maybe @kamil-tekiela has suggestions for improvement.

File: reference/mysqli/mysqli_sql_exception/getsqlstate.xml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- $Revision$ -->
+<refentry xml:id="mysqli-sql-exception.getsqlstate" xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink">
+ <refnamediv>
+  <refname>mysqli_sql_exception::getSqlState</refname>
+  <refpurpose>Gets the SQL error state</refpurpose>
```
Comment (kamil-tekiela) [inline reference/mysqli/mysqli_sql_exception/getsqlstate.xml:6]: This is incorrect. This function does not return the state of the error. It returns the [SQLSTATE](https://mariadb.com/kb/en/sqlstate/) code. The name is stylized in all caps.

File: reference/mysqli/mysqli_sql_exception/getsqlstate.xml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+ <refsect1 role="parameters">
+  &reftitle.parameters;
+  &no.function.parameters;
+ </refsect1>
+
+ <refsect1 role="returnvalues">
+  &reftitle.returnvalues;
+  <para>
+   The SQL error state.
```
Comment (kamil-tekiela) [inline reference/mysqli/mysqli_sql_exception/getsqlstate.xml:28]:
```suggestion
   Returns a string containing the SQLSTATE error code for the last error. The error code consists of five characters.
```

File: reference/mysqli/mysqli_sql_exception/getsqlstate.xml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+ </refnamediv>
+
+ <refsect1 role="description">
+  &reftitle.description;
+  <methodsynopsis>
+   <modifier>public</modifier> <type>string</type><methodname>mysqli_sql_exception::getSqlState</methodname>
+   <void/>
+  </methodsynopsis>
+  <para>
+   Gets the SQL error state.
```
Comment (kamil-tekiela) [inline reference/mysqli/mysqli_sql_exception/getsqlstate.xml:16]: Please copy the description from https://www.php.net/manual/en/mysqli.sqlstate.php or from https://www.php.net/manual/en/mysqli-stmt.sqlstate.php

**Review by kamil-tekiela — CHANGES_REQUESTED** [review]

File: reference/mysqli/mysqli_sql_exception/getsqlstate.xml [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+ <refsect1 role="description">
+  &reftitle.description;
+  <methodsynopsis>
+   <modifier>public</modifier> <type>string</type><methodname>mysqli_sql_exception::getSqlState</methodname>
+   <void/>
+  </methodsynopsis>
+  <para>
+   Returns a string containing the SQLSTATE error code for the last error.
+   The error code consists of five characters. <literal>'00000'</literal> means no error.
```
Comment (kamil-tekiela) [inline reference/mysqli/mysqli_sql_exception/getsqlstate.xml:17]:
```suggestion
   The error code consists of five characters.
```
😄

**Review by kamil-tekiela — APPROVED** [review]
LGTM Thanks!

**Pull request merged**